src/main/java/uk/gov/mca/beacons/service/mappers/AccountHolderMapper.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.mca.beacons.service.mappers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.mca.beacons.service.dto.AccountHolderDTO;
+import uk.gov.mca.beacons.service.dto.WrapperDTO;
+import uk.gov.mca.beacons.service.model.AccountHolder;
+
+@Service
+public class AccountHolderMapper extends BaseMapper {
+
+  public AccountHolder fromDTO(AccountHolderDTO accountHolderDTO) {
+    final var accountHolder = new AccountHolder();
+    accountHolder.setId(accountHolderDTO.getId());
+
+    accountHolder.setAuthId(
+      (String) accountHolderDTO.getAttributes().get("authId")
```

Comment:
Instead of calling `getAttributes` on the `accountHolderDTO`, you could set a variable of:

`final var attributes = accountHolderDTO.getAttributes()` and then call the `attributes` variable you've defined